src/Actions/ManagesContacts.php

```diff
@@ -83,6 +101,36 @@ public function findOrCreateContact($email, $firstName, $lastName, $orgid = null
         return $this->createContact($email, $firstName, $lastName, $orgid);
     }
 
+    /**
+     * Updates a contact.
+     *
+     * @param Contact|int|string $id
+     * @param string|null        $email
+     * @param string|null        $firstName
+     * @param string|null        $lastName
+     * @param null               $orgid
+     *
+     * @return Contact|null
+     */
+    public function updateContact($id, $email, $firstName, $lastName, $orgid = null)
```

Comment:
I would suggest to change this into

`updateContact(Contact $contact, array $values)`

Now you can choose what you want to update or not. When only the orgid should change, you are forced to fill in the other values also.

Reply:
I thought this too, but implemented like this to mirror `createContact`. Should I:

A] Leave as is
B] Do as you suggest
C] Do as you suggest and update `createContact` (could do some data type sniffing and in order to maintain backwards compatibility).

Reply:
@stefanius ⬆️ ?

Reply:
I will suggest ant option D, that looks like option B (with a twist):

```updateContact($id, array $values)```

Now you should always passes the $id, not a string or an object. See also my other comment on the PR.

Reply:
It is OK to differ the signature from createContact. When creating the contact there are some mandatory fields you should always should send to the API. When updating you want to be flexible.